group_vars/all.yml

```diff
@@ -0,0 +1,5 @@
+---
+provision_postgres: "true"
```

Comment:
comment:
I still find the usage of string boolean so damn ugly hehe
But I know we do it all over the place.
In fact, I don't know if this is a common thing in ansible